humilis_kinesis_proxy/resources.yaml.j2

```diff
@@ -146,8 +180,9 @@ resources:
       Type: "AWS::ApiGateway::Deployment"
       DependsOn:
         {% for resource in api_resources %}
-        - MethodPost{{resource.path.title()}}
-        - MethodOptions{{resource.path.title()}}
+        {% set resid = resource.path.title()|replace('/', '') %}
```

Comment:
Is there a way to create a function or something similar, we are repeating the replace here and if it will have to be changed it can result in hard to find bugs.